lib/utils/src/graph/algorithms.cc

```diff
@@ -21,6 +21,10 @@ std::unordered_set<Node> get_nodes(IGraphView const &g) {
   return g.query_nodes({});
 }
 
+std::unordered_set<Node> get_nodes(GraphView const &g) {
+  return g.unsafe()->query_nodes({});
```

Comment:
```suggestion
  return g.query_nodes({});
```
Also, `get_nodes(IGraphView const &)` should be removed--algorithms should now function on `Graph`s, not `IGraph`s